ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -114,10 +114,20 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     # Build a source code
     meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
 
-    # Install a nnstreamer library and examples
+    # Install a nnstreamer library
```

Comment:
With these meson options, it does not install the examples (custom filters in nnstreamer reposotiry)

Reply:
Acked.